src/uu/more/src/more.rs

```diff
@@ -105,29 +105,51 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         while let (Some(file), next_file) = (files_iter.next(), files_iter.peek()) {
             let file = Path::new(file);
             if file.is_dir() {
-                terminal::disable_raw_mode().unwrap();
-                return Err(UUsageError::new(
-                    1,
-                    format!("{} is a directory.", file.quote()),
-                ));
+                if length == 1 {
+                    terminal::disable_raw_mode().unwrap();
+                    return Err(UUsageError::new(
```

Comment:
could you please add tests to arrive in this error ? thanks

Reply:
Sure i hope i didn't miss more. grcov didnt want to work last time i tried...